server/configManager.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2016 Google Inc.
```

Comment:
I was thinking that "config" would strictly be for proto definitions representing the configuration state for the mixer.